docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -0,0 +1,145 @@
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/flexDirection.png">
+</td>
+</tr>
+</table>
+
+| Property value |  Description |
+|----------------|--------------|
+| `Row`          | Place items horizontally in a row |
+| `RowReverse`   | Place items horizontally in a row, but in reverse order |
+| `Column`       | Place items vertically in a column |
+| `ColumnReverse` | Place items vertically in a column, but in reverse order |
+
+### Usage
+
+```csharp
+View layoutView = new View();
+var flexLayout = new FlexLayout();
+flexLayout.Direction = FlexLayout.FlexDirection.Column;
+layoutView.Layout = flexLayout;
+```
+
+
+## FlexJustification
+
+`Justification` property specifies the alignment for flex items, when they do not use all available space on the main axis.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/justifyContent.png">
+</td>
+</tr>
+</table>
+
+| Property value  |   Description |
+|-----------------|---------------|
+| `FlexStart`     | Position items at the beginning of the container |
+| `FlexEnd`       | Position items at the end of the container |
+| `Center`        | Position items at the center of the container |
+| `SpaceBetween`  |  Position items with equal spacing between them |
+| `SpaceAround`   |  Position items with equal spacing before, between, and after them |
+
+### Usage
+
+```csharp
+View layoutView = new View();
+var flexLayout = new FlexLayout();
+flexLayout.Justification = FlexLayout.FlexJustification.SpaceBetween;
+layoutView.Layout = flexLayout;
+```
+
+
+## AlignmentType
+
+`Alignment` property specifies the alignment for flex items when they do not use all the available space on the cross axis.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/alignItems.png">
+</td>
+</tr>
+</table>
+
+|  Property value  |   Description  |
+|------------------|----------------|
+|  `Auto`          | Inherits the same alignment from the parent |
+|  `FlexStart`     | Align items to the beginning of the container |
+|  `Center`        | Align items to the center of the container |
+|  `FlexEnd`       | Align items to the end of the container |
+|  `Stretch`       | Stretch items to fit the container |
+
+### Usage
+
+```csharp
+View layoutView = new View();
+var flexLayout = new FlexLayout();
+flexLayout.Alignment = FlexLayout.AlignmentType.Center;
+layoutView.Layout = flexLayout;
+```
+
+
+## FlexWrapType
+
+`WrapType` property specifies whether the flex items must wrap if there is not enough room for them on 1 flex line.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/flexWrap.png">
+</td>
+</tr>
+</table>
+
+
+| Property value |  Description  |
+|----------------|---------------|
+|  `NoWrap`      | Reduce item sizes to fit them in a single line along the main axis |
+|  `Wrap`        | Show items over multiple lines, if needed |
+
+### Usage
+
+```csharp
```

Comment:
Suggestion:

Kindly add some introduction before the code block.

Reply:
Done.